package.json

```
@@ -1,19 +1,17 @@
 {
   "private": true,
   "workspaces": [
-    "site",
-    "client",
-    "website",
-    "tools"
+    "packages/*"
   ],
   "scripts": {
-    "test": "yarn test:site && yarn test:website",
-    "test:client": "yarn --cwd client test",
-    "test:site": "yarn --cwd site test",
-    "test:website": "yarn --cwd website test",
-    "build:client:docs": "yarn --cwd client typedoc",
-    "build:website": "yarn --cwd website build",
-    "format": "prettier --write '**/*.{js,jsx,ts,tsx,md,html,css}' --ignore-path .gitignore"
+    "test": "run-p test:*",
+    "test:client": "yarn --cwd packages/client test",
+    "test:api": "yarn --cwd packages/api test",
+    "test:website": "yarn --cwd packages/website test",
+    "build:client:docs": "yarn --cwd packages/client typedoc",
+    "build:website": "yarn --cwd packages/website build",
+    "format": "prettier --write '**/*.{js,jsx,ts,tsx,md,html,css}' --ignore-path .gitignore",
```

I pushed a change to main to add a `.prettierignore` as it was trying to reformat the docs directory which is not gitignored.

```suggestion
    "format": "prettier --write '**/*.{js,jsx,ts,tsx,md,html,css}'",
```